workflows/prognostic_c48_run/runtime/logs.py

```diff
@@ -83,6 +85,42 @@ def capture_fv3gfs_funcs():
         setattr(wrapper, func, captured_stream(getattr(wrapper, func)))
 
 
+def setup_logger(name):
+    logger = logging.getLogger(name)
+    fh = logging.FileHandler(f"{name}.txt")
+    fh.setLevel(logging.INFO)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    ch.setFormatter(logging.Formatter(fmt="%(levelname)s:%(name)s:%(message)s"))
+
+    logger.addHandler(fh)
+    logger.addHandler(ch)
+
+
+def setup_loggers():
+    setup_logger("statistics")
+    setup_logger("profiles")
```

Comment:
Maybe there's a way to reduce the definition/reference separation of these loggers or else warn/error when the user requests a logger not already setup.  Currently if the logger is misspelled or not specified up here, it will initialize with defaults, which is probably okay, but maybe we don't want that for consistency's sake.

Maybe we add an error when the user requests an unitialized logger (by `setup_loggers`), or just use a module-level get_logger function to ensure consistency and get rid of the `setup_loggers` function.

```
LOGGERS = {}

def setup_logger(name):
    logger = ...
    LOGGERS[name] = logger
    return logger

def get_logger(name):
    if name not in LOGGERS:
        # either error here if we want to notify user of unspecified logger
       raise Error( "you didn't add this to setup_loggers")
       # or do away with the "setup_loggers" and just trigger setup here, e.g.,
        logger = setup_logger(name)
    else:
        logger = LOGGERS[name]

    return logger

def log_values(...)
    logger = get_logger(name)
    logger.info(msg)
    ....
```

Reply:
Something like this could also be extended to the stream capturing function above to standardize w/in this module

Reply:
With this suggestion we should have a few tests to make sure the setup/get_logger are working

Reply:
Nice, yah the disconnect between the setup and logging functions is not ideal, although it is already existing before this PR.

I do think the captured logs are distinct, since they just go to stdout and don't have a file handler associated with them (the piping to `logs.txt` is defined by the parent `runfv3` script).

Perhaps if the `log_scalar` and `log_profiles` functions take a logger name, this will be cleaner.

Reply:
> I do think the captured logs are distinct, since they just go to stdout and don't have a file handler associated with them (the piping to logs.txt is defined by the parent runfv3 script).

Got it, thanks for clarifying this.